wcomponents-theme/unbuilt.package.json

```diff
@@ -18,6 +18,8 @@
 		"handlebars": "~4.0.5",
 		"i18next": "~3.4.1",
 		"i18next-xhr-backend": "~1.0.1",
```

Comment:
This is a bit sub-optimal as it may result in us breaking Intern at some point. We cannot comment a JSON file (because JSON is rubbish) but we need to put a note somewhere for whoever upgrades Intern in the future. I do not have a good solution to this.